src/main/java/trade/wayruha/whitebit/dto/Fee.java

```diff
@@ -0,0 +1,44 @@
+package trade.wayruha.whitebit.dto;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+
+@Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Fee {
+    @JsonAlias("is_depositable")
+    private boolean isDepositable;
+    @JsonAlias("is_withdrawal")
+    private boolean isWithdrawal;
+    private String ticker;
+    private String name;
+    private String[] providers;
+    private Withdraw withdraw;
+    private Deposit deposit;
+
+    @Data
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public static class Withdraw {
+        private String max_amount;
```

Comment:
we should store floating point numbers as BigDecimal (they CAN be deserialized from String)